fontforge/splinesave.c

```diff
@@ -2074,7 +2112,7 @@ struct pschars *SplineFont2ChrsSubrs(SplineFont *sf, int iscjk,
 	dummynotdef.parent = sf;
 	dummynotdef.layer_cnt = sf->layer_cnt;
 	dummynotdef.layers = calloc(sf->layer_cnt,sizeof(Layer));
-	dummynotdef.width = SFOneWidth(sf);
+	dummynotdef.width = CIDOneWidth(sf);
```

Comment:
If we're using this function in these updated cases, should the final function name be SFOneWidth again?